src/objects/entities/Bob3.ts

```diff
@@ -245,7 +246,12 @@ class Ben3 extends Entity {
 
     this.headScale = newScale;
 
-    if (this.health === 0) this.scene.scene.start('death-scene');
+    if (this.health === 0) {
+      this.scene.scene.pause();
+      setTimeout(() => {
+        this.scene.scene.start('death-scene');
+      }, 3_000);
+    }
```

Comment:
> This branch seems to lag out when you die and game changes scene. tho looks like we've got some conflicts so hoping they're changes to fix this? see what happens after conflicts resolved

is because i added a 3 second delay, happy to remove it if you want

Reply:
Feels like the game lags out, maybe we could shorten it or play a game over sound during this time

Reply:
added a 1second game over sfx and reduced the time to 1s